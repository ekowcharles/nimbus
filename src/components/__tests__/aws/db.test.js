import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import DB from '../../aws/db'

describe('DB', () => {
  it('renders databases', () => {
    const wrapper = shallow(
      <DB
        db={
          {
            "account_id": "111111111111",
            "availability_zone": "us-east-1c",
            "id": "terraform-20200121195838741200000001",
            "region": "us-east-1",
            "status": "available",
          }
        }
        key="0"
        settings={
          {
            "hide": false,
            "hideName": false,
            "hideStatus": false,
          }
        }
      />
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})