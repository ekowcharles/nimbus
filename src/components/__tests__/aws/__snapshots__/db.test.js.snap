// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DB renders databases 1`] = `
<Resource
  hideName={false}
  hideStatus={false}
  image="db.png"
  name="...000001"
  state="available"
  updateResourceAttributes={[Function]}
/>
`;
